src/SilentEchoScript.ts

```diff
@@ -86,8 +86,9 @@ export class SilentEchoScript {
         return sequences;
     }
 
-    public execute(scriptContents: string): Promise<ISilentEchoValidatorResult> {
-        return this.silentEchoValidator.execute(this.tests(scriptContents));
+    public execute(scriptContents: string): Promise<any> {
+        return this.silentEchoValidator.execute(this.tests(scriptContents),
+            this.detectInvocationName(scriptContents));
```

Comment:
The invocation name check should be per sequence....does that make sense?

Reply:
Yes, it should be per sequence, we are handling it on SilentEchoValidator side, since it handles the actual processing of the sequences, here: https://github.com/bespoken/silent-echo-sdk/pull/38/files#diff-12297744cc2151ea22fa93665e1f366aR38

Reply:
I miss-read to *auth check per sequence*, it makes sense to detect the invocation name per sequence :+1: , I'll update it.